templates/partials/scripts.hbs

```diff
@@ -28,9 +28,9 @@
 <script src='/assets/js/docs.js'></script>
 <script src='/assets/js/ga.js'></script>
 <script src='/assets/js/device-animation.js'></script>
+<script src='/assets/js/firmwareversions.js'></script>
```

Comment:
Stefan and I got that worked out @jme783 so now the script will only load on reference pages and only run on the firmware.md page.